README.md

```diff
@@ -56,4 +56,4 @@ services:
 
 ## Caveats
 
-* This recipe won't work with versions of Solr before `solr:8`, and Acquia and Pantheon.io hosting require versions from 3 to 7. You'll want to see the [contributed recipes](https://github.com/ddev/ddev-contrib) for older versions of solr.
+* This recipe won't work with versions of Solr before `solr:8`, and Acquia's hosting require versions from 3 to 7. You'll want to see the [contributed recipes](https://github.com/ddev/ddev-contrib) for older versions of solr.
```

Comment:
```suggestion
* This recipe won't work with versions of Solr before `solr:8`, and Acquia's hosting requires versions from 3 to 7. You'll want to see the [contributed recipes](https://github.com/ddev/ddev-contrib) for older versions of solr.
```

Could you take a quick look to see what Acquia's current situation with solr is?

Reply:
```suggestion
* This recipe won't work with versions of Solr before `solr:8`, and Acquia's hosting requires Solr 7. You'll want to see the [contributed recipes](https://github.com/ddev/ddev-contrib) for older versions of solr.
```